testplan/testing/multitest/base.py

```diff
@@ -397,7 +397,11 @@ def get_test_context(self):
             ) < len(sorted_testcases):
                 testcases_to_run = sorted_testcases
 
-            if self.cfg.testcase_report_target:
+            if self.cfg.testcase_report_target and not (
+                getattr(self, "parent", None)
+                and hasattr(self.parent, "cfg")
+                and self.parent.cfg.interactive_port is not None
+            ):
```

Comment:
Copied over the condition so that possible refactorization catches both occurrences, there should be better ways to test interactivity.